{
  "uid" : "dfc610334c41096b",
  "name" : "burns the requested amount",
  "fullName" : "Contract: ERC20Burnable burn when the given amount is not greater than balance of the sender for a zero amount.burns the requested amount",
  "historyId" : "caff3806dc32fc2c45d891955d1d6c89",
  "time" : {
    "start" : 1635782720253,
    "stop" : 1635782721214,
    "duration" : 961
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17ded00>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17ded00>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC20/extensions/ERC20Burnable.test.js:15:45)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ERC20Burnable burn when the given amount is not greater than balance of the sender for a zero amount"
  }, {
    "name" : "testClass",
    "value" : "Contract: ERC20Burnable burn when the given amount is not greater than balance of the sender for a zero amount"
  }, {
    "name" : "testMethod",
    "value" : "burns the requested amount"
  }, {
    "name" : "package",
    "value" : "Contract: ERC20Burnable burn when the given amount is not greater than balance of the sender for a zero amount"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "9e369183bebe3303",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238bfd32e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782602973,
        "stop" : 1635782603536,
        "duration" : 563
      }
    }, {
      "uid" : "d120876f5d838987",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a2875250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782601244,
        "stop" : 1635782601807,
        "duration" : 563
      }
    }, {
      "uid" : "f90fad692f3656b6",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a068ed60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782600839,
        "stop" : 1635782601400,
        "duration" : 561
      }
    }, {
      "uid" : "c7827004fcc29754",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042c6a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782574854,
        "stop" : 1635782575414,
        "duration" : 560
      }
    }, {
      "uid" : "b1ae383f874c5a6",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0420310>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782571985,
        "stop" : 1635782572961,
        "duration" : 976
      }
    }, {
      "uid" : "3be65a35f5ba1320",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0443910>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782522436,
        "stop" : 1635782522997,
        "duration" : 561
      }
    }, {
      "uid" : "a5541c2af96c353d",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067b700>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782510589,
        "stop" : 1635782511157,
        "duration" : 568
      }
    }, {
      "uid" : "2aac7c04de3b91cd",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07ca0a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782508092,
        "stop" : 1635782508657,
        "duration" : 565
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "30f782070c4e42dc",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/30f782070c4e42dc",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590176430,
          "stop" : 1635590188615,
          "duration" : 12185
        }
      }, {
        "uid" : "46dea5f2e9a36e9c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/46dea5f2e9a36e9c",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c4f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635554993456,
          "stop" : 1635554994444,
          "duration" : 988
        }
      }, {
        "uid" : "f0535f2742d77e99",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/f0535f2742d77e99",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551254776,
          "stop" : 1635551254852,
          "duration" : 76
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "dfc610334c41096b.json",
  "parameterValues" : [ ]
}